### ISSUE TITLE
TabControl does not render Bindable content

### ISSUE THREAD

**Issue by marcelojmacedo** [UXDivers/Grial-UI-Kit-Support#200]
Hi I am trying to implement a ListView inside a TabControl but it is not working as expected.
I am using a MVVM approach to bind the list, if I put the simple label outside the TabControl it renders but if it is inside the TabControl it does not.
Any idea what could be causing this problem?

That is an example of what I have done so far:
 
        <tab:TabControl
							TabStripPlacement="Bottom"
							StyleClass="iOSTabControlStyle">

            <tab:TabItem 
								Text="List" 
								Icon="comedy_icon.png">

                <Label Text="{Binding HeaderText}" TextColor="Black" FontSize="Large" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand"/>

            </tab:TabItem>

        </tab:TabControl>

                <Label Text="{Binding HeaderText}" TextColor="Black" FontSize="Large" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand"/>

**Comment by LeoHere** [comment]
@marcelojmacedo I was able to reproduce the issue. We will investigate it a came back to you. In the meantime, as a workaround you might do:

~~~
<ContentPage
    xmlns="http://xamarin.com/schemas/2014/forms" 
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" 
    xmlns:tab="clr-namespace:UXDivers.Artina.Shared;assembly=UXDivers.Artina.Shared.Tab"
    x:Class="..." x:Name="Root">

    <tab:TabControl ...>
        <tab:TabItem ...>
            <ListView                             
                BindingContext="{ Binding Source={x:Reference Root}, Path=BindingContext}"
                CachingStrategy="RecycleElement"
                ItemsSource="{ Binding Items }">

                <ListView.ItemTemplate>
                    <DataTemplate>
                        <ViewCell>
                            <Label Text="{Binding Name}"/>
                        </ViewCell>
                    </DataTemplate>
                </ListView.ItemTemplate>
            </ListView>
        </tab:TabItem>
    </tab:TabControl>
</ContentPage>    
~~~

**Comment by marcelojmacedo** [comment]
Thanks for your reply, it is very nice to know that we have support to your controls.
I will try it on, 🥇

**Comment by muhaym** [comment]
Can the Tab Control Text be binded?

**Comment by dirivero** [comment]
Hi @muhaym,

This problem should be fixed in version 2.5.2. You can try updating artina nuget packages to their latest version.
